src/Pretzel.Tests/Minification/LessTransformTests.cs

```diff
@@ -178,6 +178,7 @@ public void Should_Process_Less_Imports_With_Cleanup()
             Assert.False(fileSystem.File.Exists(@"c:\css\style.less"));
             Assert.True(fileSystem.Directory.Exists(@"c:\css\subfolder"));
             Assert.True(fileSystem.File.Exists(@"c:\css\subfolder\anothersubfolder\anything.less"));
+            Assert.Contains(@"c:\_site", fileSystem.AllDirectories);
```

Comment:
I think that this is what is failing the test.
Since html isn't processed and all less files outputs in `c:\css`, `c:\_site` is not created.

Reply:
But it did before, see line 182. I just used `Contains` to see what's on disk.

Reply:
So either the previous assertion was wrong or we fixed a bug that left over empty directories 👌

Reply:
I removed the assertions, cause I also think they are wrong

Reply:
Sorry, I have missed that line.
I think that the assertion was wrong and there a bug fixed in `MockFileSystem`, anyway, all good now.